goodreads_scraping/parsing/AuthorDatesParser.py

```diff
@@ -0,0 +1,32 @@
+from datetime import datetime
+from lxml import etree
+from typing import Tuple
+
+from .BaseParser import BaseParser
+
+
+class AuthorDatesParser(BaseParser):
+    def __init__(self, html):
+        self.xml = etree.fromstring(html)
+
+    def get_xpath(self, event_name: str) -> str:
+        return (
+            "//div[@class='mainContentContainer']"
+            + "//div[@class='mainContentFloat']"
+            + "//div[@class='rightContainer']"
+            + "//div[@class='dataItem' and @itemprop='{}']".format(event_name)
+        )
+
+    def parse(self) -> Tuple[datetime, datetime] | Tuple[datetime, None] | Tuple[None, None]:
+        return self._parse_helper("birthDate"), self._parse_helper("deathDate")
+
+    def _parse_helper(self, event_name: str) -> datetime:
```

Comment:
1. Please try to avoid names like "xyz_helper", unless they are really necessary (e.g. it's hard to say what a method does). Here it's clear that the method is parsing a date from a `<div>` that has a specific value in the `itemprop` attribute. So you could name it like `_parse_date_from_the_chosen_div` (just a suggestion - if you have a better idea, feel free to use it 🙂).
2. This method can also return `None`, so please update the type-hint.

Reply:
1 and 2 Agreed